tests/ImageSharp.Sandbox46/ImageSharp.Sandbox46.csproj

```diff
@@ -20,9 +20,9 @@
   <ItemGroup>
     <PackageReference Include="BitMiracle.LibJpeg.NET" Version="1.4.280" />
     <PackageReference Include="xunit" Version="2.3.1" />
-    <PackageReference Include="Moq" Version="4.8.2" />
-    <!--<PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.0.0" />-->
-    <!--<PackageReference Include="xunit.runner.visualstudio" Version="2.2.0" />-->
+    <PackageReference Include="Moq" Version="4.8.3" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.7.2" />
```

Comment:
Do we need these references? They were commented out.

Reply:
no we do not need them, i have removed it again